packages/wallet/jest.setup.js

```diff
@@ -0,0 +1,18 @@
+// Polyfill for Array.prototype.findLast on unit tests
+if (!Array.prototype.findLast) {
+  // eslint-disable-next-line no-extend-native
+  Array.prototype.findLast = function (predicate, thisArg) {
+    if (this === null) {
```

Comment:
```suggestion
    if (! (this instanceof Array)) {
```
Could this make the check a bit stronger?